### DISCUSSION TITLE
Use podman for hosting Jenkins

### DISCUSSION THREAD

**Discussion by kawaegle** [containers/podman#18993] [Q&A]
I really love Podman and almost all my software I use on my server is running as containers.

but I can't find the way to run Jenkins and add the podman socket to the jenkins clouds module so I use A podman in podman.

for now I use a jenkins docker with access to docker socket and so access to docker API and launch some build on specific Images.

I really want to do the same thing with podman but I'm not sure if it's firstly possible, and also, if it's secure (I guess because as podman don't need to be root to start containers)

If you have some idea or a way to look 

for the moment I've tried 

```docker-compose
version: '2'

services:
  jenkins:
    image: jenkins/jenkins:latest
    container_name: jenkins
    restart: always
    user: jenkins
    ports:
      - 8080:8080
      - 50000:50000
    volumes:
      - jenkins:/var/jenkins_home
      - /var/run/user/1000/podman/podman.sock:/var/run/user/1000/podman/podman.sock
volumes:
  jenkins:
```

and with docker I used 

```docker-compose
version: '2'

services:
  jenkins:
    image: jenkins/jenkins:latest
    container_name: jenkins
    privileged: true
    restart: always
    user: root
    ports:
      - 8080:8080
      - 50000:50000
    volumes:
      - jenkins:/var/jenkins_home
      - /var/run/docker.sock:/var/run/docker.sock

volumes:
  jenkins:
```
(sorry if there are gramatical mistake or english error I'm not fluent)

**Comment by kawaegle** [comment]
I guess by the fact that jenkins don't have a plugin for podman I'm not sure if it will be possible at a time podman can be used as alternative to docker in jenkins 

I left it open if someone have an idea otherwise I let someone close it

**Comment by Luap99** [comment]
You would need to mount it as `/var/run/user/1000/podman/podman.sock:/var/run/docker.sock` I believe.

**Comment by kawaegle** [comment]
It seem to work finally Thanks you, I can close the discussions I guess
